app/src/main/java/com/kickstarter/models/CheckoutPayment.kt

```diff
@@ -0,0 +1,3 @@
+package com.kickstarter.models
+
+data class CheckoutPayment(val id: Long?, val paymentUrl: String?)
```

Comment:
Nitpick here, the ID is mandat
<img width="274" alt="Screenshot 2024-02-01 at 9 06 11 AM" src="https://github.com/kickstarter/android-oss/assets/4083656/7ccf6adb-a828-4a59-8d82-a2b22de5bddb">
ory, should not be optional

Reply:
I was marking it nullable because the decode function is nullable, but ill throw an exception if its null instead.